packages/schematics/angular/migrations/update-17/use-application-builder.ts

```diff
@@ -68,8 +68,30 @@ export default function (): Rule {
           options['polyfills'] = [options['polyfills']];
         }
 
-        if (typeof options['outputPath'] === 'string') {
-          options['outputPath'] = options['outputPath']?.replace(/\/browser\/?$/, '');
+        let outputPath = options['outputPath'];
+        if (typeof outputPath === 'string') {
+          if (!/\/browser\/?$/.test(outputPath)) {
+            // TODO: add prompt.
+            context.logger.warn(
+              `The output location of the browser build have been updated from "${outputPath}" to ` +
+                `"${join(outputPath, 'browser')}". ` +
+                'You might need to adjust your deployment pipeline or, as an alternative, ' +
+                'set outputPath.browser to "" in order to maintain the previous functionality.',
+            );
+          } else {
+            outputPath = outputPath.replace(/\/browser\/?$/, '');
+          }
+
+          options['outputPath'] = outputPath;
+          if (typeof options['resourcesOutputPath'] === 'string') {
+            const media = options['resourcesOutputPath'].replaceAll('/', '');
```

Comment:
Were `media`, `media/`, `/media`, and `/media/` considered equivalent for `resourcesOutputPath`?

I think this would also treat `med/ia` as being equivalent to `media`.  Although i think that's unlikely to occur.

Reply:
Yes, all the above were considered the same.